src/Components/Form.js

```diff
@@ -33,11 +42,36 @@ function Form(props) {
       };
     });
   };
+  const validateSubmit = (e) => {
+    let isValidated = true;
+    Object.keys(formData).forEach((key) => {
+      if (formData[key] == "") {
+        setEmptyData((prev) => {
+          return {
+            ...prev,
+            [key]: true,
+          };
+        });
+        isValidated = false;
+      }
+    });
+    return isValidated;
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const isValidated = validateSubmit();
+    if (isValidated == true) {
+      const formDataCopy = { ...formData };
```

Comment:
you can add the unique ID here...
```
import { v4 as uuidv4 } from 'uuid'; // AT THE TOP OF THE FILE

const uniqueId = uuidv4();
const formDataCopy = { ...formData, id: uniqueId };
```

This way you can fetch a particular record to delete or update it later.